### DISCUSSION TITLE
Store other than collection like string, single object, int?

### DISCUSSION THREAD

**Discussion by Sadmansamee** [isar/isar#52] [Help] [answered]
Is it possible to store data types(Single object, string, int) other than @Collection? like Hive

**Comment by simc** [comment, marked as answer]
No it's not possible but you can create wrapper types like this:

```dart
@Collection()
class StringValue {
  int? id;

  String? value;
}
```

**Reply by frestoinc** [reply]
@leisim may i check what's the reason for not supporting primitive?
